lib/active_model_serializers.rb

```diff
@@ -51,6 +51,7 @@ def silence_warnings
 require 'active_model/serializer'
 require 'active_model/serializable_resource'
 require 'active_model/serializer/version'
+require 'active_model_serializers/test/serializer'
```

Comment:
Oh, and https://github.com/rails-api/active_model_serializers/pull/1270#discussion_r47167126